### PR TITLE
WIP: SIMDPolynomials support

### PR DESCRIPTION
**Pull request by YingboMa**
Don't merge this yet. SIMDPolynomials.jl hasn't been released.

### CODE REVIEW

File: src/io.jl [hunk 1]
```diff
@@ -140,6 +140,84 @@ function convert_to_internal(
...
+    ring = extract_ring(orig_polys)
+    exps, cfs = extract_polys(ring, orig_polys)
+    assure_ordering!(ring, ordering, exps, cfs)
+    ring, exps, cfs
+end
+
+function extract_ring(orig_polys::Vector{<:SIMDPolynomials.AbstractPolynomial})
+    f = first(orig_polys)
+
+    nv = maximum(SIMDPolynomials.nvariables, orig_polys)
```
Comment (sumiya11) [inline src/io.jl:159]: I am not sure if taking maximum is correct. What if the system is [x, y] ?

Comment (YingboMa) [reply src/io.jl:159]: Yeah, it's correct only for PackedMonomial, and not true in general.

**Comment by sumiya11** [comment]
Hi @YingboMa , is it true that 
https://github.com/YingboMa/SIMDPolynomials.jl/pull/5
should be merged before this one ?

**Comment by YingboMa** [comment]
Yes, indeed. I need to update this PR as well.

**Comment by sumiya11** [comment]
Cool, I'll stay tuned !

**Comment by sumiya11** [comment]
Feel free to reopen.

**Pull request closed without merging**